aiops/opni-opensearch-update-service/opensearch-update-service/app/main.py

```diff
@@ -23,10 +23,11 @@
     "ctx._source.anomaly_predicted_count += 1; ctx._source.opnilog_anomaly = true;"
 )
 
-async def doc_generator(df):
+async def doc_generator(df, index, op_type="index"):
```

Comment:
default op_type should be `update`.

Reply:
By default, if _op_type isn't specified, it's index.